content/en/data_streams/_index.md

```diff
@@ -57,9 +59,15 @@ To get started, follow the installation instructions to configure services with
 
 ## Explore Data Streams Monitoring
 
+### Visualize the architecture of your streaming data pipelines
+
+{{< img src="data_streams/topology_map.png" alt="A DSM topology map visualization. " style="width:100%;" >}}
+
+Data Streams Monitoring provides a [topology map][10], so that you can visualize interdependencies, service ownership, and key health metrics across the services and infrastructure your pipeline depends on.
```

Comment:
Data Streams Monitoring provides an out-of-the-box topology map so you can visualize data flow across your pipelines, and easily identify producer/consumer services, and queue dependencies, along with service ownership and key health metrics.